### DISCUSSION TITLE
[FEATURE]在openeuler环境下，没有pure-ftpd 和fail2ban

### DISCUSSION THREAD

**Discussion by liuze0109** [1Panel-dev/1Panel#6713] [Ideas]
### 1Panel 版本

v1.10.18-lts

### 请描述您的需求或者改进建议

官网的安装文档，不适用于openeuler 环境。可以考虑其他的兼容程序比如vsftpd 等替代。
https://1panel.cn/docs/user_manual/toolbox/ftp/
https://1panel.cn/docs/user_manual/toolbox/fail2ban/
如果是确实没有方案，可以判断操作系统是openeuler的话就在工具箱页面不显示这 2 个菜单
![iShot_2024-10-14_14 59 11](https://github.com/user-attachments/assets/6db8aebc-2161-4399-8212-14149830dbe1)

**Comment by wanghe-fit2cloud** [comment]
文档中提供的方案确实不适用于 OpenEuler 操作系统：
1. 使用兼容程序的方式并推荐，因为这样做会造成修改量过大。
2. 虽然可以实现隐藏菜单，但并不推荐。建议通过 dnf 或源码编译的方式安装 pure-ftpd 和 fail2ban。可以参考相关的安装教程来完成。
